v3/fleet-local/control/cfn-signal@.service

```diff
@@ -16,7 +16,10 @@ ExecStartPre=/usr/bin/systemctl is-active zk-health.service
 ExecStartPre=/usr/bin/systemctl is-active mesos-master@*
 ExecStartPre=/usr/bin/systemctl is-active marathon@*
 ExecStartPre=/usr/bin/sh -c "docker pull $($IMAGE)"
-ExecStartPre=/usr/bin/sh -c "/usr/bin/docker run --rm\
+ExecStartPre=/usr/bin/sh -c "/usr/bin/docker run \
+--log-opt max-size=10m \
```

Comment:
should this be ` --log-opt max-size=`etcdctl get /docker/config/logs-max-size` ?